modules/test/playwright/pages/users-admin-web/AccountSettingsPage.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {Locator, Page} from '@playwright/test';
+
+export class AccountSettingsPage {
+	readonly accountSettingsMenuItem: Locator;
+	readonly page: Page;
+	readonly userPersonalMenuButton: Locator;
+	readonly rolesMenuItem: Locator;
+	readonly saveButton: Locator;
+
+	constructor(page: Page) {
+		this.accountSettingsMenuItem = page.getByRole('menuitem', {
+			name: 'Account Settings',
+		});
+		this.userPersonalMenuButton = page.getByTestId('userPersonalMenu');
```

Comment:
same here, probably now they are in wrong order because we renamed this variable